tests/integration/test_session.py

```diff
@@ -327,3 +327,29 @@ def test_create_spark_session_with_session_template_and_user_provided_dataproc_c
         Session.State.TERMINATED,
     ]
     assert DataprocSparkSession._active_s8s_session_uuid is None
+
+
+def test_add_artifacts_pypi_package():
```

Comment:
I think that we need to parametrize it at least w/ `image_version` so it runs against all supported runtimes?

Reply:
It is yet to be supported in 3.0, in 2.2 python version is different, so those tests would fail, hence only running with default 2.3 now.